_targets.R

```diff
@@ -24,19 +26,29 @@ major_ion_names = c("Chloride","Sodium")
 # Define hydrologic event types in harmonized WQP data to exclude
 omit_wqp_events <- c("Spill","Volcanic action")
 
-# Define USGS specific conductance parameter codes
-SpC_pcodes <- c("00095","90095","00094","90096") 
+# Define all USGS specific conductance parameter codes
+pcodes_select <- c("00095","90095","00094","90096") 
```

Comment:
Perhaps I shouldn't say "all" on line 29 here. I originally went through and found all of the SC pcodes but ultimately decided to omit the more extraneous ones and narrow down to this target list that represent both in situ and lab-measured SC values. No DRB sites get returned from any of these other pcodes so I am going to leave `pcodes_select` as is. However, I'll make a note of these other pcodes in the markdown doc since I've been tracking some data processing decisions there.

Reply:
Sounds good, thanks!